.github/workflows/docker_release.yml

```
@@ -19,12 +19,20 @@ jobs:
       - name: Set env
         id: set_tag_version
         run: |
-          if [ "${{ github.event.client_payload.tagversion }}" != "" ]; then
-          echo ::set-output name=tagversion::${{ github.event.client_payload.tagversion }}
+          #Checks for a valid tag when the tag is manually pushed
+          if [[ $GITHUB_REF != "" && "${{ github.event.client_payload.tagversion }}" == "" ]]; then
+          echo ::set-output name=tagversion::$(egrep -o '(v[0-9]+\.[0-9]+\.[0-9]+)(-.*)?' <<< $GITHUB_REF)
```

lets make sure it only matches against non whitespace characters
```suggestion
          echo ::set-output name=tagversion::$(egrep -o '(v[0-9]+\.[0-9]+\.[0-9]+)(-\S*)?' <<< $GITHUB_REF)
```

---

i've applied the commit suggestions but it seems that it wasn't picked up. I'll push the branch. Thank you for catching this